models/five_fold_validation.py

```diff
@@ -0,0 +1,41 @@
+from logistic_regression import LogisticRegression
+import pandas as pd
+import numpy as np
+
+
+def validate_logistic_regression_for_wine_quality():
+    num_of_folds = 5
+    learning_rate= 0.000001
+    max_iterations = 100
+    df = pd.read_csv("../data/winequality/winequality-red.csv", sep=";")
+    df['classified']=[1 if x>=6 else 0 for x in df["quality"] ]
+    fold_size = int(round(df.shape[0]/num_of_folds))
+    for i in range(5):
+        x_test = df[['fixed acidity', 'volatile acidity', 'citric acid', 'residual sugar', 'chlorides', 'free sulfur dioxide', 'total sulfur dioxide', 'density', 'pH', 'sulphates', 'alcohol']][i*fold_size:fold_size+i*fold_size]
+        x_train_part_1 = df[['fixed acidity', 'volatile acidity', 'citric acid', 'residual sugar', 'chlorides', 'free sulfur dioxide', 'total sulfur dioxide', 'density', 'pH', 'sulphates', 'alcohol']][fold_size+i*fold_size:]
+        x_train_part_2 = df[['fixed acidity', 'volatile acidity', 'citric acid', 'residual sugar', 'chlorides', 'free sulfur dioxide', 'total sulfur dioxide', 'density', 'pH', 'sulphates', 'alcohol']][:i*fold_size]
+        x_train = x_train_part_1.append(x_train_part_2)
```

Comment:
this is what I am talking about, like why do you need x_train_part_1 and x_train_part_2, when you are simply combining them at the end.

Reply:
say we have taken rows 320 to 640 for validation. then the training set is 0:319 + 641:1598.

Reply:
oh k, got it

Reply:
To be more precise training set is rows 641:1598 + 0:319